src/debugger/Debugger.cpp

```diff
@@ -499,6 +499,38 @@ bool Debugger::processIncomingMessages(ExecutionState* state, ByteCodeBlock* byt
 
     while (receive(buffer, length)) {
         switch (buffer[0]) {
+        case ESCARGOT_DEBUGGER_CLIENT_SOURCE: {
+            StringBuilder builder;
+            uint32_t size;
+            char* data;
+            memcpy(&size, buffer + 1, sizeof(uint32_t));
+            data = (char*)GC_MALLOC_ATOMIC(size);
+            char* ptr = data;
+            char* end = data + size;
+            length = ESCARGOT_DEBUGGER_MAX_MESSAGE_LENGTH - 1 - sizeof(uint32_t);
+            memcpy(ptr, buffer + 1 + sizeof(uint32_t), length);
+            ptr += length;
+            do {
+                if (receive(buffer, length)) {
+                    memcpy(ptr, buffer + 1, length - 1);
```

Comment:
We should check the message type here for sanity check. I think the method we used for eval (has a start and a continuation message) is a bit better since it allows further checks.